api/src/test/resources/TestingApplicationContext.xml

```diff
@@ -19,6 +19,11 @@
         <property name="mappingJarLocations">
             <ref bean="mappingJarResources"/>
         </property>
+        <property name="packagesToScan">
+            <list>
+                <value>org.openmrs</value>
+            </list>
+        </property>
```

Comment:
What is the reason for this?  (Again, just curious so I can understand the changes)

Reply:
Required for testing against OpenMRS 2.x I believe.  This isn't a 2.5 thing, but a 2.x thing, and we had only been running tests up until now against 1.10.  I believe this allows the test framework to pick up hibernate mappings that are defined as annotations rather than as hbm.xml files.